turtlebot_navigation/launch/includes/gmapping/gmapping.launch.xml

```diff
@@ -3,7 +3,7 @@
   <arg name="base_frame"  default="base_footprint"/>
   <arg name="odom_frame"  default="odom"/>
 
-  <node pkg="gmapping" type="slam_gmapping" name="slam_gmapping" output="screen">
+  <node pkg="nodelet" type="nodelet" name="slam_gmapping" args="load SlamGMappingNodelet camera/camera_nodelet_manager" output="screen">
```

Comment:
People use the turtlebot gmapping with Lidars, in which case they would not have the camera nodelet manager.
I am not entirely sure that it is a good idea to make loading the nodelet the default. 

I think it would be better to make a separate laser gmapping launch, that does not use the nodelet, before merging,